site/src/content/docs/components/buttons.mdx

```diff
@@ -17,15 +17,28 @@ import { getMode } from '@libs/sass-variables'
 You can find here the [OUDS Button design guidelines](https://unified-design-system.orange.com/472794e18/p/48a788-button).
 </Callout>
 
-## Base class
+## Overview
 
 OUDS Web has a base `.btn` class that sets up basic styles such as padding and content alignment. By default, `.btn` controls have a transparent border and background color, but still defines basic focus and hover styles.
 
 <Example code={`<button type="button" class="btn">Base class</button>`} />
 
-The `.btn` class is intended to be used in conjunction with our button variants, or to serve as a basis for your own custom styles.
+The `.btn` class is intended to be used in conjunction with our button variants, or to serve as a basis for your own custom styles. The most basic button is the `.btn-default` variant.
 
-## Standard variants
+<Example code={`<button type="button" class="btn btn-default">Default</button>`} />
+
+The `.btn` classes are designed to be used with the `<button>` element. However, you can also use these classes on `<a>` or `<input />` elements (though some browsers may apply a slightly different rendering).
```

Comment:
Not a big fan to have this in here since we show variants before explaining them. Imo we can move this button tags part inside Layout part.